Wrappers/Python/cil/optimisation/operators/MatrixOperator.py

```diff
@@ -22,27 +22,39 @@
 from cil.optimisation.operators import LinearOperator
 
 class MatrixOperator(LinearOperator):
-    """ Matrix wrapped into a LinearOperator
+    r""" Matrix wrapped in a CIL Operator to be used in optimisation algorithms. 
 
-    :param: a numpy matrix
+    Parameters      
+    ----------
+    A: a numpy matrix
+        The matrix to be wrapped into a CIL Operator
 
     """
 
     def __init__(self,A):
-        '''creator
-
-        :param A: numpy ndarray representing a matrix
-        '''
+        """Constructor"""
         self.A = A
         M_A, N_A = self.A.shape
         domain_geometry = VectorGeometry(N_A, dtype=A.dtype)
         range_geometry = VectorGeometry(M_A, dtype=A.dtype)
-        self.s1 = None   # Largest singular value, initially unknown
         super(MatrixOperator, self).__init__(domain_geometry=domain_geometry,
                                                    range_geometry=range_geometry)
 
     def direct(self,x, out=None):
-
+        r"""Returns :math:`Ax`
+        
+        Parameters
+        ----------
+        x : DataContainer
+            Input data
+        out : DataContainer, optional
+            Output data, default is None
```

Comment:
Should we add that:

If out is not None the output of the Operator will be filled in out, otherwise a new object is instantiated and returned.